YSI_Players/y_text/y_text_impl.inc

```diff
@@ -569,6 +569,8 @@ static stock Text_LoadLocals()
 
 stock _Text_CheckOwnership(file[], str[])
 {
+	#pragma unused file
+	#pragma unused str
 	// TODO...
```

Comment:
These warnings are useful, since the code isn't ported yet.  Supressing them hides work to be done.